packages/swirl-components/src/components/swirl-tab/swirl-tab.stories.ts

```diff
@@ -24,6 +24,7 @@ export const SwirlTab = Template.bind({});
 
 SwirlTab.args = {
   active: true,
+  icon: '<swirl-icon glyph="emoji-satisfied" size="20"></swirl-icon>',
```

Comment:
You can use the ´<swirl-tab-bar>` component to show a standalone tab bar. The tab bar is configured via JSON. Or you could use the `<swirl-tabs>` with nested `<swirl-tab>` components.

**Angular with standalone tab bar:**
```html
<swirl-tab-bar label="My tabs" [tabs]="[{ "active": false, "id": "tab1", "label": "Tab #1" }, { "active": true, "icon": "<swirl-icon glyph=\"emoji-mood\" size=\"20\"></swirl-icon>", "id": "tab2", "label": "Tab #2" }]"></swirl-tab-bar>
```

This renders just the tab bar. You would have to implement your own tab navigation logic via the component's events.

**Angular with swirl-tabs:**
```html
<swirl-tabs initial-tab="tab-2" label="Tabs">
    <swirl-tab icon="<swirl-icon-emoji-satisfied></swirl-icon-emoji-satisfied>" label="Tab #1" tab-id="tab-1">. 
        <swirl-text>Tab 1 content</swirl-text>
    </swirl-tab>
    <swirl-tab icon="<swirl-icon-emoji-mood></swirl-icon-emoji-mood>" label="Tab #2" tab-id="tab-2">
        <swirl-text>Tab 2 content</swirl-text>
    </swirl-tab>
</swirl-tabs>
```

This renders a tab bar and the tabs with content. Also, takes care of the tab navigation logic.

Reply:
Thanks for the explanation. I thought the preview in the storybook is a bit confusing, since you don't see the tab and only the content string. But then again there is the reference to the SwirlTabs component, so it's fine for me. 

![swirl-tab](https://github.com/getflip/swirl/assets/142502606/c265552c-0f37-49ff-b69e-2b2bfbf4454a)